### ISSUE TITLE
Assertion Errors in `test_plot` 

### ISSUE THREAD

**Issue by def-mycroft** [Graeme22/tastyworks-cli#19]
After building the virtual environment and running `make test`, I receive [these errors][1]. 

Below is the error section:

```
=============================================== FAILURES ================================================
_____________________________________ test_twcli_plot_pandl_percent _____________________________________

    def test_twcli_plot_pandl_percent():
        pf = gen_portfolio()
        pf_tmp = gen_portfolio(True)
        nl = pf_tmp._get_starting_net_liq('1y')
        val = pf.plot('1y', starting_net_liq=nl)
    
>       assert round(val, 2) == _REALIZED_PANDL_PERCENT
E       assert -13.61 == -14.56
E        +  where -13.61 = round(-13.612839563968665, 2)

tests/test_plot.py:42: AssertionError
____________________________________ test_twcli_plot_netliq_percent _____________________________________

    def test_twcli_plot_netliq_percent():
        pf = gen_portfolio(True)
        pf_tmp = gen_portfolio(True)
        nl = pf_tmp._get_starting_net_liq('6m')
        val = pf.plot('6m', starting_net_liq=nl)
    
>       assert round(val, 2) == _NET_LIQUIDITY_PERCENT
E       assert 89.78 == 87.41
E        +  where 89.78 = round(89.77612117172752, 2)

tests/test_plot.py:51: AssertionError
```

[1]:https://paste.ubuntu.com/p/d6JCsXjHtZ/

**Comment by Graeme22** [comment]
Yup, I accidentally made the tests non-deterministic. Fixed by #18.

**Comment by Graeme22** [comment]
#18 is merged, so closing this.
